src/components/Items/Scorecard/Scorecard.vue

```diff
@@ -0,0 +1,252 @@
+<template>
```

Comment:
`Items` was initially thought to be for items like question or note or others that creator can create. I can see how scorecard fits into it. In that case, if scorecard further doesn't have any sub-components (like question had header, body, footer), then, let's only keep `Items/Scorecard.vue`?